hivemq-edge/src/frontend/src/locales/en/components.json

```diff
@@ -61,6 +61,15 @@
     "action_COPIED": "Copied!",
     "action_ERROR": "Error!"
   },
+  "PrivacyConsentBanner": {
+    "header": "Privacy Settings",
+    "body": "This web application uses third-party website tracking technologies to provide and continually improve our services. I agree and may revoke or change my consent at any time with effect for the future.",
```

Comment:
I find the last sentence a little confusing.
Proposal for an alternative: "This web application uses third-party tracking technologies to continually improve our services. You can agree to their use now and change or revoke your consent at any time in the future."